### DISCUSSION TITLE
unable to verify Discord

### DISCUSSION THREAD

**Discussion by 15414h** [chromiecraft/chromiecraft#3034] [General]
your discord channel wants my phone number for verification, so i cant join.

**Comment by 55Honey** [comment]
Hello 15414h,
Our discord server is set to the highest protection level, which requires a discord account verified with a phone number. Unfortunately, there is no workaround for this.

We however don't see any phone numbers. People verify their discord Account against their phone number, it's not related to our server specifically.

Once verified, your discord account can make use of it on all servers.

**Reply by 15414h** [reply]
thanks. I cant give my phone number to Discord either. Guess I have to miss out on whatever your Discord offers.
